README.md

```diff
@@ -182,3 +182,21 @@ the API documentation in [toxcore/tox.h](toxcore/tox.h) for more information.
 
 - [Another echo bot](https://wiki.tox.chat/developers/client_examples/echo_bot)
 - [minitox](https://github.com/hqwrong/minitox) (A minimal tox client)
+
+## SAST Tools
+
```

Comment:
Maybe add a short sentence before the list, like "List of static analysis tools used by this project:" or "This project is checked by the following static analysis tools:" or something along these lines?

Reply:
Since the rest of README is talking about Tox/Toxcore , it's just a bit confusing as to why this section is in the README, how is it Tox-related? Is this a list of tools that use toxcore? Is this a list of tools that toxcore provides when built? So on. Perhaps the confusion is made worse when you don't know what SAST is. So just adding a sentence like this should make things abundantly clear, otherwise it's a bit cryptic if you are not in the know.

Reply:
Added a sentence.